File: Sources/BibleVerseReplacer/AppInfo.swift
import Foundation

enum AppInfo {
    static let repositoryURLString = "https://github.com/maxiaovo/Bible-Verse-Replacer"
    static let repositoryURL = URL(string: repositoryURLString)!

    static var version: String {
        Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String ?? "0.0.0"
    }

    static var build: String {
        Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as? String ?? "0"
    }

    static var versionDisplay: String {
        "v\(version) (\(build))"
    }
}

File: Windows/BibleVerseReplacer.Windows/AppInfo.cs
using System;
using System.Reflection;

namespace BibleVerseReplacer.Windows
{
    internal static class AppInfo
    {
        public const string RepositoryUrl = "https://github.com/maxiaovo/Bible-Verse-Replacer";

        public static string Version
        {
            get
            {
                System.Version version = Assembly.GetExecutingAssembly().GetName().Version;
                return version.Major + "." + version.Minor + "." + version.Build;
            }
        }

        public static string VersionDisplay
        {
            get { return "v" + Version; }
        }
    }
}
